### PR TITLE
Fix bug 1120619 - Restart all celery components during deploy. Refs bug 1122085.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

File: scripts/chief_deploy.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def prime_app(ctx):
 @hostgroups(settings.CELERY_HOSTGROUP, remote_kwargs={'ssh_key': settings.SSH_KEY})
 def update_celery(ctx):
     ctx.remote(settings.REMOTE_UPDATE_SCRIPT)
-    ctx.remote('/usr/bin/supervisorctl restart %s' % settings.CELERY_SERVICE)
+    ctx.remote('/usr/bin/supervisorctl mrestart celery*')
```
Comment (groovecoder) [inline scripts/chief_deploy.py]: `mrestart` typo? Or are we using a library or something that gives it to us?

Comment (jezdez) [inline scripts/chief_deploy.py]: no typo, we use a supervisor plugin that adds wildcard support via an own command

Comment (jezdez) [inline scripts/chief_deploy.py]: https://pypi.python.org/pypi/supervisor-wildcards/

**Comment by jezdez** [comment]
Okay, cyliang gave the go on Bugzilla for stage: https://bugzilla.mozilla.org/show_bug.cgi?id=1120619

**Pull request merged**